server/src/main/java/com/linecorp/centraldogma/server/internal/api/converter/QueryRequestConverter.java

```diff
@@ -72,13 +77,15 @@ private static String getPath(ServiceRequestContext ctx) {
         return "";
     }
 
-    private static Optional<String> getExpression(ServiceRequestContext ctx) {
+    private static Optional<Iterable<String>> getExpressions(ServiceRequestContext ctx) {
         final String query = ctx.query();
         if (query != null) {
-            final List<String> expression = new QueryStringDecoder(query, false).parameters().get(
-                    "expression");
-            if (expression != null) {
-                return Optional.of(expression.get(0));
+            final List<String> queryExpressions = new QueryStringDecoder(query, false).parameters().get(
+                    "expressions");
+            if (queryExpressions != null) {
+                final String[] expressions = SPLIT.split(queryExpressions.get(0));
```

Comment:
- How about letting a user specify multiple `expression` parameters? e.g. `expression=exprA&expression=exprB&...` This would allow us to handle the case where an expression contains a comma.
- How about renaming the parameter to `jsonpath`, just in case we introduce more expression types in the future? i.e. `jsonpath=exprA&jsonpath=exprB&...`

Reply:
All fixed. Thank you~!